presto-tests/src/test/java/io/prestosql/tests/TestTpchDistributedStats.java

```diff
@@ -41,7 +42,9 @@ public void setup()
     {
         DistributedQueryRunner runner = TpchQueryRunnerBuilder.builder()
                 // We are not able to calculate stats for PARTIAL aggregations
-                .amendSession(builder -> builder.setSystemProperty(PREFER_PARTIAL_AGGREGATION, "false"))
+                .amendSession(builder -> builder
+                        .setSystemProperty(PREFER_PARTIAL_AGGREGATION, "false")
```

Comment:
the comment `// We are not able to calculate stats for PARTIAL aggregations` should be moved too